app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._conection = sqlite3.connect(r'D:\Games\cinema.db')
```

Comment:
Consider parameterizing the database path or moving it to a configuration file for better maintainability and flexibility.